stable/kommander/templates/grafana/opsportal-username-secret.yaml

```diff
@@ -19,7 +19,7 @@ spec:
       containers:
         - name: kubectl
           # --export flag is deprecated so we need to stick with this kubectl version
-          image: bitnami/kubectl:1.16.2
+          image: bitnami/kubectl:1.19.7
```

Comment:
let me check into this one, we are using the `--export` flag which is deprecated in newer kubectl versions